sass/_base.scss

```diff
@@ -21,4 +35,4 @@ body {
 .container {
   max-width: 1280px;
   margin: 0 auto;
-}
```

Comment:
check closing curly brackets "}"

Reply:
The closing curly bracket is there, the error is something else